.github/ISSUE_TEMPLATE/crx-tut-request.yaml

```diff
@@ -0,0 +1,61 @@
+name: Extension tutorial request
+description: Use this form to request a new Chrome Extension tutorial
+title: "[EXT-TUTORIAL] <title>"
+labels: [content, extensions]
+assignees: amysteam
+body:
+- type: markdown
+  attributes:
+    value: "## Thank you for sharing your feedback. Please fill out this form so that we can better understand what extension tutorial you are looking for:"
+- type: textarea
+  attributes:
+    label: What would this tutorial do?
+    description: A concise description of what the tutorial should teach.
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: What lessons would this tutorial teach?
+    description: Top three extension concepts it will teach.
+    placeholder: |
+      1. ...
+      2. ...
+      3. ...    
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: What Chrome APIs should be featured?
+    description: Chrome APIs you'd like to see demonstrated.
+    placeholder: |
+      If you are not sure, please leave blank.
+  validations:
+    required: false
+- type: dropdown
+  id: components
+  attributes:
+    label: Extension components
+    description: Is there a specific component you would like to see?
+    multiple: true
+    options:
+      - Service worker
+      - Popup
+      - Overlay (content script)
+      - Omnibox
+      - Context Menu
+      - Options page
+      - Any
```

Comment:
Is there a way to add 'other' with a blank to fill in?

Reply:
That would be nice, however, I don't think the [dropdown](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema#dropdown) component supports this feature 😕